src/Symfony/Component/OptionsResolver/OptionConfigurator.php

```diff
@@ -90,21 +90,8 @@ public function define(string $option): self
      *
      * @return $this
      */
-    public function deprecated(/*string $package, string $version, $message = 'The option "%name%" is deprecated.'*/): self
+    public function deprecated(string $package, string $version, $message = 'The option "%name%" is deprecated.'): self
```

Comment:
The `OptionConfigurator` class wasn't released yet so we can still make this change.